src/config/config.ts

```diff
@@ -221,7 +230,7 @@ export class Config implements IConfig {
   async loadDevPlugins() {
     if (this.options.devPlugins !== false) {
       // do not load oclif.devPlugins in production
-      if (this.isProd) return
```

Comment:
I wonder if `isProd` should check the existence of a manifest (and/or other files) instead of just `NODE_ENV`.

not sure where that method would live (or can be a func that accepts `this.root`).

Reply:
first versions of core were checking for a manifest too, changed here: https://github.com/oclif/core/pull/129/files#diff-615dacd7000fa06d7afef0f22b7b6d9267c6219fbfc9fa0315e51a5f5d963755L186